examples/chatmemberbot.py

```diff
@@ -80,7 +80,10 @@ async def track_chats(update: Update, context: ContextTypes.DEFAULT_TYPE) -> Non
     chat = update.effective_chat
     if chat.type == Chat.PRIVATE:
         if not was_member and is_member:
-            logger.info("%s started the bot", cause_name)
+            # This is not really needed in this particular case because /start command will run
+            # after the user unblocks the bot, and the start function will add the user to
+            # "user_ids".  We're including this here for the sake of the example.
```

Comment:
```suggestion
            # "user_ids". We're including this here for the sake of the example.
```

Reply:
PEP8 [states](https://peps.python.org/pep-0008/#comments):

> You should use two spaces after a sentence-ending period in multi- sentence comments, except after the final sentence.

Reply:
wow, that's insane. TIL

Reply:
WTF